recipe/meta.yaml

```diff
@@ -8,26 +8,27 @@ package:
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/charset-normalizer-{{ version }}.tar.gz
-  sha256: f23667ebe1084be45f6ae0538e4a5a865206544097e4e8bbcacf42cd02a348f3
+  sha256: f30c3cb33b24454a82faecaf01b19c18562b1e89558fb6c56de4d9118a032fd5
 
 build:
   number: 0
-  noarch: python
+  script: {{ PYTHON }} -m pip install . --no-deps -vv --no-build-isolation
   entry_points:
-    - normalizer = charset_normalizer.cli.normalizer:cli_detect
-  script: {{ PYTHON }} -m pip install . -vv
+    - normalizer = charset_normalizer.cli:cli_detect
+  skip: True  # [py<37]
 
 requirements:
   host:
-    - python >=3.5
+    - python
     - pip
+    - setuptools
```

Comment:
@psteyer Could you try to build it with `mypy`? It will speedup `four times faster`!
```
  host: 
    - mypy >=1.6.1  # [py>38]
    - mypy >=1.4.1  # [py<38]
```
To enable this speedup, add `script_env` to the `build` section:
```
  script_env:
    - export CHARSET_NORMALIZER_USE_MYPYC=1
```

See https://github.com/jawah/charset_normalizer/blob/3.3.2/docs/community/speedup.rst and https://github.com/jawah/charset_normalizer/blob/3.3.2/build-requirements.txt#L3-L4

Reply:
I'm getting the following error when trying to make the above changes:
`ERROR: Could not build wheels for charset-normalizer, which is required to install pyproject.toml-based projects`

Reply:
@skupr-anaconda I would keep it like it is now, see https://github.com/AnacondaRecipes/charset-normalizer-feedstock/pull/1/files#r1687187823.